src/cli/README.md

```diff
@@ -33,11 +33,11 @@ OpenThread test scripts use this CLI to execute test cases.
 * [stop](#stop)
 * [whitelist](#whitelist)
 
-## OpenThread Command Details
+## OpenThread command details
 
 ### channel
 
-Get the IEEE 802.15.4 Channel value.
```

Comment:
Should terms that are defined by a protocol specification be capitalized or lower case?